tests/unit/components/atoms/AppTextInput.spec.js

```diff
@@ -0,0 +1,50 @@
+import { describe, expect, it } from '@jest/globals'
+import { shallowMount } from '@vue/test-utils'
+import AppTextInput from '@/components/atoms/AppTextInput.vue'
+
+describe('Test AppTextInput', () => {
+  const propsWithOptional = {
+    placeholder: 'placeholder',
+    maxlength: 5,
+  }
+
+  const propsWithoutOptional = {
+    placeholder: 'placeholder',
+  }
+
+  it.each([[propsWithOptional], [propsWithoutOptional]])('should render', (props) => {
+    const wrapper = shallowMount(AppTextInput, {
+      props: props,
+    })
+
+    expect(wrapper.exists()).toBe(true)
+    expect(wrapper.element).toMatchSnapshot()
+  })
+
+  it('should sets the value', async () => {
+    const wrapper = shallowMount(AppTextInput, {})
```

Comment:
You don't need to call it in each test, only once is enough. If you need to change props values, please, use setProps